src/test/java/org/pesmypetcare/webservice/dao/medalmanager/MedalDaoTest.java

```diff
@@ -0,0 +1,79 @@
+package org.pesmypetcare.webservice.dao.medalmanager;
+
+
+import com.google.cloud.firestore.DocumentSnapshot;
+import com.google.cloud.firestore.WriteBatch;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.pesmypetcare.webservice.dao.medalmanager.MedalDao;
+import org.pesmypetcare.webservice.dao.medalmanager.MedalDaoImpl;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No need to import a type that lives in the same package](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517612)